.eslintrc.cjs

```diff
@@ -1,9 +1,11 @@
 /* eslint-env node */
+require("@rushstack/eslint-patch/modern-module-resolution");
```

Comment:
What's this one for? 🤔

Reply:
No idea 😉 I have just used `.eslintrc.cjs` from Tomek's demo repo to be sure TS works fine in my cinema. Seems I can remove this one and nothing explodes 😉

Reply:
You definitely can 😄